### PR TITLE
Datastore 0.15.1

### PR DESCRIPTION
**Pull request by lukesneeringer**
A prerequisite step for https://github.com/GoogleCloudPlatform/google-cloud-python/issues/3025

### CODE REVIEW

**Comment by lukesneeringer** [comment]
I figured out while looking at this PR why this one was special/different; I had forgotten that we released an 0.15.0 for this several weeks earlier. I must not have noticed when firing through commands to release all the new ones later. (Notice how this one does not have the GAPIC `lib_name` changes.)

File: generated/python/gapic-google-cloud-datastore-v1/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-googleapis-common-protos>=1.5.0, <2.0dev
-google-gax>=0.15.4, <0.16dev
-proto-google-cloud-datastore-v1>=0.90.0, <0.91dev
```
Comment (bjwatson) [inline generated/python/gapic-google-cloud-datastore-v1/requirements.txt]: Manually fix the `proto-` version

File: generated/python/gapic-google-cloud-datastore-v1/setup.py [hunk 2]
```diff
@@ -9,15 +9,15 @@
 import sys
 
 install_requires = [
-    'googleapis-common-protos>=1.5.0, <2.0dev',
-    'google-gax>=0.15.4, <0.16dev',
-    'proto-google-cloud-datastore-v1[grpc]>=0.90.0, <0.91dev',
```
Comment (bjwatson) [inline generated/python/gapic-google-cloud-datastore-v1/setup.py]: Ditto.

File: generated/python/gapic-google-cloud-datastore-v1/requirements.txt (hunk 1 above)
Comment (bjwatson) [inline generated/python/gapic-google-cloud-datastore-v1/requirements.txt]: Also, shouldn't there be a `[grpc]` option here?

**Pull request merged**